counting_sort/counting_sort.py

```diff
@@ -0,0 +1,30 @@
+def countOccurences(a, c):
+    for i in range(len(a)):
```

Comment:
ok..I am just a beginner in python. Didn't think of optimizing at this stage. Will develop better coding styles.